### PR TITLE
refactor: delete unnecessary requiered fields

### PR DESCRIPTION
**Pull request by nidalhjab**

### CODE REVIEW

File: packages/server/src/controllers/driver/driver-controller.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ export const getDrivers = async (req: Request, res: Response) => {
 	}
 };
 export const postDriver = async (req: Request, res: Response) => {
-	const { name, lat, lng, phone, carModel, licensePlate, date, locationName, locationId, userId } = req.body;
+	const { name, phone, carModel, licensePlate, locationId, userId } = req.body;
```
Comment (marwank7) [inline packages/server/src/controllers/driver/driver-controller.ts:14]: Why did you remove locationName?

Comment (nidalhjab) [reply packages/server/src/controllers/driver/driver-controller.ts:14]: no locationName sent from FE, it's not required also, because by locaionId we can get all inforamtion

Comment (marwank7) [reply packages/server/src/controllers/driver/driver-controller.ts:14]: Oh okay nevermind, just realised it's postDriver

**Review by marwank7 — APPROVED** [review]

**Review by hamzahmurrar — APPROVED** [review]
